src/search/operators/ranking/FastNonDomSorting.ts

```diff
@@ -48,14 +48,14 @@ export function fastNonDomSorting<T extends Encoding>(
       let pDominatesQ = true;
       let qDominatesP = true;
       for (const key of objectiveFunctions) {
-        if (p.getDistance(key)! === 0 && q.getDistance(key)! === 0) {
+        if (p.getDistance(key) === 0 && q.getDistance(key) === 0) {
```

Comment:
What did the exclamation mark do?

Reply:
ensure "not null" but other changes I made, already ensure "not null"